tmt/steps/discover/fmf.py

```diff
@@ -82,6 +185,31 @@ def wake(self):
             if value:
                 self.data[option] = value
 
+    def check_requires(self, all_requires):
```

Comment:
IMO misleading - How about `check_dependencies`? And even better if 'check' can be renamed - it just detects those, no checking at all

Reply:
Yes, that method name is a challenge. During implementation I have renamed it already three times ;-) Let's call it just `dependencies` ;-)